apps/web/src/pages/templates/components/formTypes.ts

```diff
@@ -15,7 +20,7 @@ export interface ITemplates extends IMessageTemplate {
   layoutId?: string;
 }
 
-export interface IStepEntity {
+export interface IFormStep {
```

Comment:
this is the interface of the "form step", it's different from the BE interface by metadata fields